pymilo/__main__.py

```diff
@@ -0,0 +1,119 @@
+        '--compression',
+        type=str,
+        choices=['NULL', 'GZIP', 'ZLIB', 'LZMA', 'BZ2'],
+        default='NULL',
+        help='Specify the compression method (NULL, GZIP, ZLIB, LZMA, or BZ2). Default is NULL.'
+    )
+    parser.add_argument(
+        '--port',
+        type=int,
+        default=8000,
+        help='Specify PyMiloServer port number',
+        metavar="",
+    )
+    parser.add_argument(
+        '--protocol',
+        type=str,
+        choices=['REST', 'WEBSOCKET'],
+        default='REST',
+        help='Specify the communication protocol (REST or WEBSOCKET). Default is REST.'
+    )
+    parser.add_argument(
+        '--load',
+        type=str,
+        default=None,
+        help='the `load` command specifies the path to the JSON file of the previously exported ML model by PyMilo.',
+        metavar="",
+    )
+    parser.add_argument(
+        '--init',
+        type=str,
+        default=None,
+        help='the `init` command specifies the ML model to initialize the PyMilo Server with.',
+        metavar="",
+    )
+    parser.add_argument(
+        '--bare',
+        default=False,
+        action='store_true',
+        help='The `bare` command starts the PyMilo Server without an internal ML model.',
+    )
+    parser.add_argument('--version', action='store_true', default=False, help='PyMilo version')
+    parser.add_argument('-v', action='store_true', default=False, help='PyMilo version')
+    args = parser.parse_args()
+    if args.version or args.v:
+        print(PYMILO_VERSION)
+        return
+    if not ml_streaming_support:
+        print("Error: ML Streaming is not installed.")
+        print("To install ML Streaming, run the following command:")
+        print("pip install pymilo[streaming]")
+        print("For more information, visit the PyMilo README at https://github.com/openscilab/pymilo")
+        tprint("PyMilo")
+        tprint("V:" + PYMILO_VERSION)
+        pymilo_help()
+        parser.print_help()
+        return
+    if args.load:
+        path = args.load
+        model = Import(url=path) if re.match(URL_REGEX, path) else Import(file_adr=path)
+        PymiloServer(
+            model=model,
+            port=args.port,
+            compressor=Compression[args.compression],
+            communication_protocol=CommunicationProtocol[args.protocol],
+        ).communicator.run()
+    elif args.init:
+        model_name = args.init
+        model_class = get_sklearn_class(model_name)
+        if model_class is None:
+            print(
+                "The given ML model name is neither valid nor supported, use the list below: \n{print_supported_ml_models}")
+            print_supported_ml_models()
+            return
+        PymiloServer(
+            model=model_class(),
+            port=args.port,
+            compressor=Compression[args.compression],
+            communication_protocol=CommunicationProtocol[args.protocol],
+        ).communicator.run()
```

Comment:
I believe you can write these sections better. apply `if`s so that you change `model` accordingly and do `PymiloServer(...).communicator.run()` just one time.